### PR TITLE
update for 5 minute tutorial

### PR DESCRIPTION
**Pull request by khevse**
## Checklist

- [x ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [ x] I have read the [security policy](../security/policy)
- [x ] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation within the code base (if appropriate)
- [ ] I have documented my changes in the [developer guide](https://github.com/ory/docs) (if appropriate)

### CODE REVIEW

File: quickstart.yml [hunk 1]
```diff
@@ -29,14 +29,14 @@ services:
       - URLS_LOGOUT=http://127.0.0.1:3000/logout
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
-      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - IDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
```
Comment (aeneasr) [inline quickstart.yml]:
```suggestion
      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
```

File: test/e2e/docker-compose.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ services:
       - URLS_LOGOUT=http://127.0.0.1:5002/logout
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
-      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - IDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
```
Comment (aeneasr) [inline test/e2e/docker-compose.yml]:
```suggestion
      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise

```

File: test/e2e/circle-ci.env.bash [hunk 3]
```diff
@@ -5,8 +5,8 @@ export URLS_CONSENT=http://127.0.0.1:5002/consent
 export URLS_LOGIN=http://127.0.0.1:5002/login
 export URLS_LOGOUT=http://127.0.0.1:5002/logout
 export SECRETS_SYSTEM=youReallyNeedToChangeThis
-export OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-export OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+export IDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
```
Comment (aeneasr) [inline test/e2e/circle-ci.env.bash]:
```suggestion
export OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
```

Comment (khevse) [reply test/e2e/circle-ci.env.bash]: I'm sorry. It's work at night.

Comment (aeneasr) [reply test/e2e/circle-ci.env.bash]: No problem :)

File: quickstart.yml [hunk 4]
```diff
@@ -29,14 +29,14 @@ services:
...
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
+      - OIDC_SUBJECT_IDENTIFIERS_PAIRWISE_SALT=youReallyNeedToChangeThis
     restart: unless-stopped
 
   consent:
     environment:
       - HYDRA_ADMIN_URL=http://hydra:4445
-    image: oryd/hydra-login-consent-node:v1.0.0-rc.10
+    image: oryd/hydra-login-consent-node:v1.1.1
```
Comment (aeneasr) [inline quickstart.yml]:
```suggestion
    image: oryd/hydra-login-consent-node:v1.2.2
```

File: quickstart.yml [hunk 5]
```diff
@@ -29,14 +29,14 @@ services:
       - URLS_LOGOUT=http://127.0.0.1:3000/logout
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
-      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
```
Comment (aeneasr) [inline quickstart.yml]: The naming of this key was actually wrong, I've fixed the naming in #1706 . The proper key is now:

```suggestion
      - OIDC_SUBJECT_IDENTIFIERS_SUPPORTED_TYPES=public,pairwise
```

Config Key `OIDC_SUBJECT_IDENTIFIERS_ENABLED` is still valid but will show a deprecation warning.

File: test/e2e/circle-ci.env.bash [hunk 6]
```diff
@@ -5,8 +5,8 @@ export URLS_CONSENT=http://127.0.0.1:5002/consent
 export URLS_LOGIN=http://127.0.0.1:5002/login
 export URLS_LOGOUT=http://127.0.0.1:5002/logout
 export SECRETS_SYSTEM=youReallyNeedToChangeThis
-export OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-export OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+export OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
```
Comment (aeneasr) [inline test/e2e/circle-ci.env.bash]:
```suggestion
      - OIDC_SUBJECT_IDENTIFIERS_SUPPORTED_TYPES=public,pairwise
```

File: test/e2e/docker-compose.yml [hunk 7]
```diff
@@ -18,8 +18,8 @@ services:
       - URLS_LOGOUT=http://127.0.0.1:5002/logout
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
-      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
```
Comment (aeneasr) [inline test/e2e/docker-compose.yml]:
```suggestion
      - OIDC_SUBJECT_IDENTIFIERS_SUPPORTED_TYPES=public,pairwise
```

File: quickstart.yml (hunk 4 above)
Comment (khevse) [reply quickstart.yml]: [oryd/hydra-login-consent-node:v1.2.2](https://hub.docker.com/r/oryd/hydra-login-consent-node/tags) not found. 
Please push this container to hub.docker.com .

Comment (aeneasr) [reply quickstart.yml]: Ohhhh, my bad! We can leave it as is then :) I thought this was the hydra image :)

**Pull request merged**